dev/io.openliberty.mcp.internal/resources/OSGI-INF/l10n/metatype.properties

```
@@ -0,0 +1,14 @@
+mcpServerConfig.name=MCP Server Configuration
+mcpServerConfig.description=Configure the MCP Server feature.
+
+stateless.name=Stateless Mode
+stateless.description=Enable stateless mode to improve scalability by turning off session tracking and request cancellation.\n\
+This is useful when using multiple servers or load balancers where sessions cannot be shared.\n\n\
+Pros:\n\
+- Easier horizontal scaling\n\
+- Works well with cloud/serverless setups\n\
+- Less server state to manage\n\n\
+Cons:\n\
+- No session tracking\n\
+- Request cancellation is disabled\n\
+- Some tool lifecycle features may not work
```

This is good and I think you've captured all of the important information. I would suggest the following rewording which I think is a little clearer.

```suggestion
stateless.description=Stateless mode disables session tracking. This allows for easier clustering, without the load balancer needing to ensure that each session is served by a single backend server. However, it also disables MCP features which rely on session tracking or linking separate HTTP requests together, such as cancelling a tool call.
```

In the future we'll add to this message since there'll be more things that require sessions.